tests/WFC3_ecfs/S4_WFC3.ecf

```diff
@@ -0,0 +1,41 @@
+# Eureka! Control File for Stage 4: Generate Lightcurves 
+
+# Number of spectroscopic channels spread evenly over given wavelength range
```

Comment:
This file seems to be missing several parameters specified in [demos/HST/S3_wfc3_template.ecf](https://github.com/kevin218/Eureka/pull/366/files#diff-3565656719afa8751a190b2bda27fd8debe72626c64efe2a3eee49f9c77fd984)